concepticondata/conceptlists/Lionnet-1985-364.tsv

```diff
@@ -0,0 +1,365 @@
+Lionnet-1985-364-311	311	sangre	blood	946	BLOOD
+Lionnet-1985-364-312	312	sapo	toad	894	TOAD
+Lionnet-1985-364-313	313	secarse	to dry up	1786	DRY UP
+Lionnet-1985-364-314	314	seis	six	1703	SIX
+Lionnet-1985-364-315	315	sembrar	sow seeds	748	SOW SEEDS
+Lionnet-1985-364-316	316	semilla	seed	714	SEED
+Lionnet-1985-364-317	317	sentarse	sit down	1649	SIT DOWN
+Lionnet-1985-364-318	318	sentarse (suj. plur.)	sit down (several)		
+Lionnet-1985-364-319	319	sentir, desear	feel, desire	1079	FEEL
+Lionnet-1985-364-320	320	ser hondo	be deep	1593	DEEP
+Lionnet-1985-364-321	321	serpiente	snake	730	SNAKE
+Lionnet-1985-364-322	322	sol	sun	1343	SUN
+Lionnet-1985-364-323	323	soltarse	become loose, untie	1933	LOOSEN
+Lionnet-1985-364-324	324	sombra	shade, shadow	1388	SHADE
+Lionnet-1985-364-325	325	soplar	blow	176	BLOW (WITH MOUTH)
+Lionnet-1985-364-326	326	subir	go up, ascend	705	GO UP (ASCEND)
+Lionnet-1985-364-327	327	suegra	mother in-law	1050	MOTHER-IN-LAW
+Lionnet-1985-364-328	328	suegro	father in-law	1055	FATHER-IN-LAW
+Lionnet-1985-364-329	329	tabaco	tobacco	974	TOBACCO
+Lionnet-1985-364-330	330	tender	lay out horizontally, extend		
+Lionnet-1985-364-331	331	tener dificultad, ser flojo	have difficulties, be lazy		
+Lionnet-1985-364-332	332	tener miedo	be scared	1419	FEAR (BE AFRAID)
+Lionnet-1985-364-333	333	tener vergüenza	be ashamed		
+Lionnet-1985-364-334	334	teta	breast, teat	1402	BREAST
+Lionnet-1985-364-335	335	tierra	earth, soil	1228	EARTH (SOIL)
+Lionnet-1985-364-336	336	tierra de siembra	field for sowing	212	FIELD
+Lionnet-1985-364-337	337	tigre	tiger (jaguar)	846	TIGER
+Lionnet-1985-364-338	338	tirarle	throw	1456	THROW
+Lionnet-1985-364-339	339	tocar música	play music		
+Lionnet-1985-364-340	340	topo	mole	1907	MOLE
+Lionnet-1985-364-341	341	tortuga	turtle	112	TURTLE
+Lionnet-1985-364-342	342	toser	cough	879	COUGH
+Lionnet-1985-364-343	343	tostar	to toast		
+Lionnet-1985-364-344	344	tres	three	492	THREE
+Lionnet-1985-364-345	345	tripa	gut	1334	GUTS
+Lionnet-1985-364-346	346	tuna	prickly pear		
+Lionnet-1985-364-347	347	uña	nail	1896	FINGERNAIL OR TOENAIL
+Lionnet-1985-364-348	348	uno	one	1493	ONE
+Lionnet-1985-364-349	349	venado	deer	1936	DEER
+Lionnet-1985-364-350	350	venado bura	mule deer		
+Lionnet-1985-364-351	351	ver	see	1409	SEE
+Lionnet-1985-364-352	352	verde, azul	blue, green	2382	BLUE OR GREEN
+Lionnet-1985-364-353	353	verruga	wart	2665	WART
+Lionnet-1985-364-354	354	víbora de cascabel	rattlesnake	871	RATTLESNAKE
+Lionnet-1985-364-355	355	viejo	old	1229	OLD
+Lionnet-1985-364-356	356	viento	wind	960	WIND
+Lionnet-1985-364-357	357	volar	to fly	1441	FLY (MOVE THROUGH AIR)
+Lionnet-1985-364-358	358	vomitar	vomit	1278	VOMIT
+Lionnet-1985-364-359	359	yerno	son-in-law	1056	SON-IN-LAW
+Lionnet-1985-364-360	360	zancudo	mosquito	1509	MOSQUITO
+Lionnet-1985-364-361	361	zopilote	black vulture		
+Lionnet-1985-364-362	362	zorillo	skunk	3888	SKUNK
+Lionnet-1985-364-363	363	zorra	fox (female)		
+Lionnet-1985-364-364	364	dejar	leave (behind)		
```

Comment:
Could be mapped to LEAVE I think

Reply:
This is the definition of LEAVE in Concepticon: To move away from a place, a situation or a talk. I do not believe it can be mapped for "dejar".

Reply:
The definition are only examples, but it is true that without further context, it could also mean something like "permit", and we cannot disambiguate easily.